src/app/src/components/RouteNotFound.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { memo } from 'react';
+import Grid from '@material-ui/core/Grid';
+
+import AppGrid from './AppGrid';
+
+const RouteNotFound = memo(() => (
+    <AppGrid title="Route not found">
+        <Grid container className="margin-bottom-64">
+            <Grid item xs={12}>
+                <p>
+                    Route not found
```

Comment:
I would like to consider using the standard "Not found." Route is a bit technical.

Reply:
Sounds good. I pushed an adjusted commit to do this in cafc8dfe46774e8fc50e9deedc7e1e4ed40cb826